.gitignore

```diff
@@ -12,3 +11,4 @@ docs/
 npm-debug.log
 /.idea/
 .env
+dist/*
```

Comment:
That was the intent, but I don't think it works. I think dist/xterm.js still gets added when you `git add`, just not `dist/addons/*` (yet).